axlearn/common/utils.py

```diff
@@ -102,6 +105,26 @@ def sharding(self) -> jax.sharding.Sharding:
 NestedTensorSpec = Optional[Union[TensorSpec, dict[str, Any]]]
 
 
+def offload_dots_saveble(offload_src, offload_dst):
```

Comment:
Missing types/returns?

```suggestion
def offload_dots_saveable(offload_src, offload_dst):
```